src/app/error-dialog/error-dialog.component.css

```diff
@@ -0,0 +1,8 @@
+.mat-raised-button {
+  float: none;
+  margin: 0 auto;
+}
+
+.mat-dialog-title {
```

Comment:
technically it is not a class, strange that you can select this via css selector, but if it works-it works.